Client/Phpredis/ClientCluster.php

```diff
@@ -0,0 +1,1545 @@
+<?php
+
+/*
```

Comment:
You can remove these docblock entirely

Reply:
I think i didn't understand this suggestion, meantime pushed a new revision